.github/workflows/deploy.yml

```
@@ -75,12 +67,12 @@
         run: docker load < reearth-marketplace.tar.gz
       - name: docker push
         run: |
-          docker tag reearth/reearth-marketplace:nightly $IMAGE_GCP
-          docker push $IMAGE_GCP
+          docker tag reearth/reearth-marketplace-api:nightly $SERVER_IMAGE_GCP
+          docker push $SERVER_IMAGE_GCP
       - name: Deploy to Cloud Run
         run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:2:11: Double quote to prevent globbing and word splitting

(shellcheck)